recipe/meta.yaml

```diff
@@ -14,11 +14,11 @@ build:
     - gdptools = gdptools.__main__:main
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
 
 requirements:
   host:
-    - python >=3.9,<3.12
+    - python >=3.9,<3.13
```

Comment:
Metadata upstream says <=3.12, making the 3.12.5 greater than the one allowed. Poetry is super strict in those pins. See https://code.usgs.gov/wma/nhgf/toolsteam/gdptools/-/blob/develop/pyproject.toml?ref_type=heads#L36